examples/classifying_shapes.ipynb

```diff
@@ -59,7 +59,8 @@
    "outputs": [],
```

Comment:
<p>do you need to specify the solver? i think <code>lbfgs</code>  is the default </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/giotto-ai/giotto-tda/pull/462/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Reply:
<p>You are right that it is. This is an old notebook...</p>

Reply:
<p>Removed.</p><p> </p>